docs/INSTALLATION.md

```diff
@@ -1,36 +1,42 @@
 # Installation
 
-Please reference Racket's [Getting Started](http://docs.racket-lang.org/getting-started/) page for instructions on downloading and installing Racket.
+The Racket distribution can be downloaded from https://download.racket-lang.org
 
-Alternative installation options and further notes for specific OSes follow.
+Installers are provided for 
 
-## macOS
+* Windows
+* Linux
+* macOS
 
-Install via Homebrew:
+Please reference Racket's [Getting Started](http://docs.racket-lang.org/getting-started/) page for guidance materials.
```

Comment:
The INSTALLATION.md is primarily intended to help students who want to work outside of the online Exercism editor. Resources about Racket that are relevant to both students working offline and online should go in RESOURCES.md or LEARNING.md. LEARNING.md should be specific to learning Racket, but RESOURCES.md is more general so things can be in both files. I think the links not about installing Racket can go sit in RESOURCES.md if they're not there already.

Reply:
The official getting started is specifically about getting and installing racket and setting up command line tools
I don’t see how this is not appropriate?
```suggestion
Please reference Racket's [Getting Started](http://docs.racket-lang.org/getting-started/) page for guidance materials.
```
![IMG_7418](https://github.com/exercism/racket/assets/108047/8a82ce80-05e7-43c5-9086-af2c39652c71)